graphdatascience/procedure_surface/api/catalog_endpoints.py

```diff
@@ -106,3 +170,30 @@ class GraphFilterResult(BaseResult):
     node_count: int
     relationship_count: int
     project_millis: int
+
+
+class GraphGenerationStats(BaseResult):
+    name: str
+    nodes: int
+    relationships: int
```

Comment:
this deviates from our naming convention
```suggestion
    node_count: int
    relationship_count: int
```

Reply:
Nevermind, that's what the proc API does. Sigh.

Reply:
I could rename them, but this is consistent with the proc API